docs/user-guide-policy-configure-ovs-bridge.md

```diff
@@ -0,0 +1,86 @@
+# Tutorial: Create a Open vSwitch Bridge and Connect it to a Node Interface
+
+Use Node Network Configuration Policy to configure a new ovs bridge `br1` connected
+to node interface `eth1`.
+
+## Requirements
+
+Before we start, please make sure that you have your Kubernetes/OpenShift
+cluster ready. In order to do that, you can follow the guides of deployment on
+[local cluster](deployment-local-cluster.md) or your
+[arbitrary cluster](deployment-arbitrary-cluster.md).
+
+You must also be sure that [OpenVSwitch](https://www.openvswitch.org/) and [NetworkManager OpenVSwitch plugin](https://developer.gnome.org/NetworkManager/stable/nm-openvswitch.html) are installed in the nodes.
+
+Please also note that on OpenShift `openvswitch` is already installed as a daemon, so only `nm-openvswitch` is required.
+
+## Configure bridge
+
+All you have to do in order to create the bridge on all nodes across cluster is
+to apply the following policy:
+
+```yaml
+cat <<EOF | kubectl create -f -
+apiVersion: nmstate.io/v1alpha1
+kind: NodeNetworkConfigurationPolicy
+metadata:
+  name: br1-eth1-policy
+spec:
+  desiredState:
+    interfaces:
+      - name: br1
+        description: ovs bridge with eth1 as a port
+        type: ovs-bridge
+        state: up
+        bridge:
+          options:
+            stp: false
+          port:
+            - name: eth1
+EOF
+```
+
+You can also remove the bridge with following:
+
+```yaml
+cat <<EOF | kubectl create -f -
+apiVersion: nmstate.io/v1alpha1
+kind: NodeNetworkConfigurationPolicy
+metadata:
+  name: br1-eth1-policy
+spec:
+  desiredState:
+    interfaces:
+      - name: br1
+        type: ovs-bridge
+        state: absent
+EOF
+```
```

Comment:
Please add section explaining that while the previous example may be enouh/useful to provide bridge for Pods via OVS CNI, it won't have any IP endpoint in the system. To do so, internal iface has to be created (and show how).

Reply:
I would not speak about the pod connectivity but keep talking about the host networking. After all, this may be used just to configure the host. Using the ovs with the pod is the next step.

Reply:
Fair enough, but please explain the difference between the two.